### DISCUSSION TITLE
Lua highlight is broken after the comment

### DISCUSSION THREAD

**Discussion by DifferentialOrange** [github-linguist/linguist#6676] [Classification]
Lua syntax local keyword highlight is broken for a line if preceding line is a comment. Reproduces both on Mozilla Firefox and Google Chrome for Ubuntu, as well as Google Chrome for Android.

Examples:

![image](https://github.com/github-linguist/linguist/assets/20455996/67d47f56-38fe-4b71-aa4e-fc31cc155739)
https://github.com/tarantool/vshard/blob/5bfe03b48839863c7a0098d27dbbee1d54890562/vshard/storage/exports.lua#L6

![image](https://github.com/github-linguist/linguist/assets/20455996/ae1d0ed9-211b-49d8-8939-d7ae6884c3a8)
https://github.com/tarantool/tarantool/blob/1fe7186779b1314fc50a7725be023ec1d845b48c/src/box/lua/config/applier/sharding.lua#L6

**Comment by DifferentialOrange** [comment]
`function` also breaks up, so it's safe to assume that next keyword is always broken

![image](https://github.com/github-linguist/linguist/assets/20455996/33f109b9-cb09-4491-bab4-d5f46bdcd8b4)
https://github.com/tarantool/tarantool/blob/1a5e3bf3c3badd14ffc37b2b31e47554e4778cde/test/luatest_helpers/server.lua#L185

**Comment by lildude** [comment]
This is likely to be the pinned known issue: https://github.com/github-linguist/linguist/issues/6668
